File: TODOList-Android/src/main/java/com/easy/todolist/android/enum/TaskCategory.kt
package com.easy.todolist.android.enum

enum class TaskCategory(
    val color: Long,
    val category: String
) {
    WORK(color = 0xFFF79E89, category = "WORK"),
    LIFE(color = 0xFFF76C6A, category = "LIFE"),
    GAMING(color = 0xFFF58B70, category = "GAMING")
}

File: TODOList-iOS/TODOList-iOS/enum/TaskCategory.swift
//
//  TaskCategory.swift
//  TODOList-iOS
//
//  Created by Jin on 2023/8/11.
//  Copyright © 2023 orgName. All rights reserved.
//

import Foundation

enum TaskCategory: CaseIterable {
    case WORK
    case LIFE
    case GAMING
}

func getCategory(category: TaskCategory) -> Int64 {
    return switch category {
    case .WORK: 0xFFF79E89
    case .LIFE: 0xFFF76C6A
    case .GAMING: 0xFFF58B70
    }
}
